presto-parquet/src/main/java/com/facebook/presto/parquet/writer/ParquetWriterOptions.java

```diff
@@ -22,6 +22,7 @@ public class ParquetWriterOptions
 {
     private static final DataSize DEFAULT_MAX_ROW_GROUP_SIZE = DataSize.valueOf("128MB");
     private static final DataSize DEFAULT_MAX_PAGE_SIZE = DataSize.valueOf("1MB");
+    private static final DataSize DEFAULT_MAX_DICTIONARY_PAGE_SIZE = DEFAULT_MAX_PAGE_SIZE;
```

Comment:
Is there a reason to have a separate default variable here if it is just reference to the other field? It's also not exactly clear to me what the implications are of having these two values equivalent. Personally I think it would be more clear to remove this line and explain why the defaults are the same in the builder (See other review comment).

Reply:
To set the two separate default values equivalent is just to keep consistent with parquet-mr [https://github.com/apache/parquet-mr/blob/9b5a962df3007009a227ef421600197531f970a5/parquet-column/src/main/java/org/apache/parquet/column/ParquetProperties.java#L49](url). And I agree with your point that it would be more clear to have only one default value and add some comment to explain. I will fix it.